docs/1_USING_COBALT.md

```diff
@@ -0,0 +1,13 @@
+# 1. Using and Customizing Cobalt
+
+[Documentation WIP] A series of hands-on labs that helps a user of Cobalt understand how to effectively create, test, and operationalize an infrastructure template ("stack").
```

Comment:
IMO, the concept of the "lab" should be expressed in our documentation but explicitly using the term  "lab" feels academic, like a readme for a class room assignment. Would "mamuals" be better here?

Reply:
let's leave out lab/manual etc... KISS (Keep it Simple)